### PR TITLE
Oy2 8208

### PR DESCRIPTION
**Pull request by daniel-belcher**
Link to ticket: https://qmacbis.atlassian.net/browse/OY2-8208?atlOrigin=eyJpIjoiNTRlMWIzYjc5YWU4NGYxZjkwN2NlYWUxMWY0NjlkMjgiLCJwIjoiaiJ9

Link to instance: https://d38jivs84rr16u.cloudfront.net

#### Pull Request Creator Checklist

- [x] This PR has an associated issue or issues.
- [x] The associated issue(s) are linked above.
- [x] This PR meets all acceptance criteria for those issues.
- [x] This PR and linked issue(s) are adequately documented
- [x] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.
- [x] Someone has been assigned this PR.
- [x] At least one person has been marked as reviewer on this PR.

#### Pull Request Reviewer/Assignee Checklist

- [ ] This PR has an associated issue or issues.
- [ ] The associated issue(s) are linked above.
- [ ] This PR meets all acceptance criteria for those issues.
- [ ] This PR and linked issue(s) are adequately documented
- [ ] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.

### CODE REVIEW

**Review by jdinh8124 — COMMENTED** [review]
@daniel-belcher Did we fix those bugs during the demo this morning? Otherwise, I can give this the ok

File: services/ui-src/src/components/MonthPicker/calendarPopup.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  "August",
+  "September",
+  "October",
+  "November",
+  "December",
+];
+
+export const MonthPickerCalendar = ({
+  selectedMonth,
+  selectedYear = "2021",
```
Comment (benjaminpaige) [inline services/ui-src/src/components/MonthPicker/calendarPopup.tsx:33]: can we pull this default value from config.currentReportingYear theres a config file at the src root

File: services/ui-src/src/components/MonthPicker/calendarPopup.tsx [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+                icon={<BsChevronRight />}
+                onClick={() => {
+                  setYear((pr) => pr + 1);
+                }}
+              />
+            )}
+          </CUI.HStack>
+        </CUI.PopoverHeader>
+        <CUI.PopoverBody>
+          <CUI.SimpleGrid columns={3}>
```
Comment (benjaminpaige) [inline services/ui-src/src/components/MonthPicker/calendarPopup.tsx:95]: great use of simplegrid : )

File: services/ui-src/src/components/MonthPicker/index.tsx [hunk 3]
```diff
@@ -1,117 +1,92 @@
...
-];
+
+type YearProps =
+  | {
+      yearLocked?: false;
+      initYear?: string;
+    }
+  | { yearLocked: true; initYear: string };
+
+type Props = CommonProps & YearProps;
```
Comment (benjaminpaige) [inline services/ui-src/src/components/MonthPicker/index.tsx:17]: 👍

**Review by benjaminpaige — DISMISSED** [review]
one small request, but otherwise it looks great!

**Comment by benjaminpaige** [comment]
Merged in master and updated default year from my comment above.

**Pull request merged**